admin/views/pupil-status/pupil-status-details/error-details.ejs

```diff
@@ -0,0 +1,37 @@
+<details class="govuk-details govuk-!-margin-bottom-0" data-module="govuk-details">
+    <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+          Steps to take
+        </span>
+    </summary>
+    <div class="govuk-details__text">
+        <dl class="govuk-panel-list">
+            <dt>
+                <div><strong><u>Either</u> log into the device that pupil used</strong></div>
+                <div>
+                    <p>
+                        Step 1 <br/>
+                        Log into the device this pupil used to take the check and the Live MTC should send through automatically.
+                    </p>
+                    <p>
+                        Step 2 <br/>
+                        Check this <u>pupil status</u> page to see if this pupil is now listed as 'Complete'. If not follow the instructions below
+                    </p>
+                </div>
+            </dt>
+            <dt>
+                <div><strong><u>or</u> restart check for pupil</strong></div>
```

Comment:
The `u` tag is now the Unarticulated Annotation Element and NOT underline.  I don't see why we would want to underline anyway - won't this get mistaken for a link?